design/FY2020/NFP-ZoneAirTransferModel.md

```diff
@@ -0,0 +1,143 @@
+Zone Air Mass Flow Balance Improvement
+=====================================
+
+**Bereket Nigusse, FSEC/UCF**
+
+ - Original Date: January 19, 2021
+ - Revision Date: none
+ 
+
+## Justification for New Feature ##
+
+ZoneAirMassFlowConservation is intended to balance airflows but current design is that the procedure adjusts zone mixing and/or infiltration rates. User that have complex air transfer models like grocery stores, restaurants, and some big box retailers need the ability to model actual air transfer and account for ventilation movement between spaces. In these cases, users want the infiltration, zone mixing, and exhaust fan flow rates to stay fixed at user specified flows, instead adjust the system return air to balance zone air flow. 
```

Comment:
How does the proposed method model or give user control over transfer air?

Reply:
Users will specify transfer air using ZoneMixing objects and these flow rates remain the same.

Reply:
OK, but with this method, the ZoneMixing flows will occur no matter what the system flows are. It will be up to the user to schedule the ZoneMixing flows to match the HVAC system operation.

Reply:
Yes, that is my plan.

Reply:
I will update the NFP based on the feedback and seek input from those requested the feature.  @mjwitte and @rraustad Thank you for your review and feedback.